src/index.js

```diff
@@ -3,8 +3,35 @@
  * @return {number[]}
  */
 
+
 const pascalTriangle = (lineNumber) => {
-  
+	// Declaramos dos arreglo, uno el que retornaremos y el otro es un arreglo auxiliar
+	let triangle = [1];
+	let triangleAux = [];
+
+	// Consideramos un caso y regresamos solo el [1]
+	if (lineNumber <= 0)
+		return triangle
+
+	// Consideramos como entrada el número 1 y devolvemos [1, 1]
+	triangle.push(1);
+
+	// Consideramos los casos en que la entrada sea 2 o más
+	var N = 2;
+	for(let j=1; j<lineNumber; j++) {
```

Comment:
No usas j en el bloque for. Por lo que te recomiendo usar j en lugar de N, acomodando los límites `let j =2; j<= lineNumber`